test/Base.t.sol

```diff
@@ -138,7 +138,7 @@ abstract contract Base_Test is Assertions, Events, StdCheats, StdUtils {
     }
 
     /// @dev Conditionally deploy the registry either normally or from a source precompiled with `--via-ir`.
-    function deployRegistryConditionally() internal {
+    function deployRegistryConditionally() private {
```

Comment:
This function may be called by future contracts inheriting `Base`, so keeping it on `internal` is fine.

Reply:
If by "future contracts" you're referring to the tests for the next version of `PRBProxy`, then, I don't think this is a valid reason to keep the function `internal`. If my understanding is correct, the next version of the project should have its separate test suite (which could be partially copied from the current one, but would, still, be a different set of tests, generally speaking).

Reply:
Development would continue in the same repo (see https://github.com/PaulRBerg/prb-proxy/issues/89). Also, this is a minor point and we're in a hurry to get to launch - it's fine to keep it as `internal`